listeners.go

```diff
@@ -495,6 +495,21 @@ func ParseNetworkAddress(addr string) (NetworkAddress, error) {
 	}, nil
 }
 
+// SplitNetwork splits a into a network, if any, and rest.
+func SplitNetwork(a string) (network, rest string, err error) {
+	beforeSlash, afterSlash, slashFound := strings.Cut(a, "/")
+	if slashFound {
+		network = strings.ToLower(strings.TrimSpace(beforeSlash))
+		a = afterSlash
+	}
```

Comment:
The ToLower and TrimSpace probably aren't necessary (or shouldn't be, anyway)

Reply:
I just copied this from SplitNetworkAddress below. If you think these shouldn’t be necessary, can I ask you to clean them up in a separate change? I’d like to keep the scope of my change to a bare minimum.

Reply:
Sure. And I can probably remove them from below too, just never really thought about it.